src/codeu/chat/client/View.java

```diff
@@ -305,7 +305,8 @@ public User getSignInStatus(String name, String password){
 	  System.out.println("ERROR: Exception during call on server. Check log for details.");
 	  LOG.error(ex, "Exception during call on server.");
       }
-
+      if (user == null)
```

Comment:
nit: in Java please add {} for any if/for/else/etc statements. 
Also user == null does not always mean that the password is incorrect, right? It could also be that the user name is not found.